hbase-server/src/main/java/org/apache/hadoop/hbase/master/HMaster.java

```diff
@@ -1908,14 +1907,18 @@ public RegionNormalizer getRegionNormalizer() {
     return this.normalizer;
   }
 
+  public boolean normalizeRegions() throws IOException {
+    return normalizeRegions(new NormalizeTableFilterParams.Builder().build());
```

Comment:
Since this is being called by chore continuously, maybe we can create a constant and use it?

Reply:
Keeping an object forever in OldGen vs creating and GC'ing the object... for a chore that runs every 30 minutes? (shrug) If you feel strongly, sure.

Reply:
Yeah you are right, GC'ing sounds better. Thanks.